jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
I agree, it will be overcomplication to keep both solutions. The question is which is easier and feels more natural for the user. Maybe we should ask some people who are not involved, what would they prefer to do? (choose from a list or provide their own data from the beginning?)